docker-compose.yml

```diff
@@ -1,15 +1,15 @@
 version: "3"
 services:
-  flask:
-    container_name: oracle
-    image: oracle-image
+  crypto-oracle:
+    container_name: crypto-oracle
```

Comment:
Except having strong reason to have fixed container name, I suggest let docker compose generate its name for us. This way we will not have name collision when run more than one instance. We may or may not need to do that but not preventing ourselves to be able to do that is always better.